src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/Cas2ApplicationEntity.kt

```diff
@@ -25,6 +28,14 @@ interface Cas2ApplicationRepository : JpaRepository<Cas2ApplicationEntity, UUID>
 SELECT
     CAST(a.id AS TEXT) as id,
     a.crn,
+    a.noms_number as nomsNumber,
```

Comment:
I wonder why we have to list these all out instead of *? As we're using a summary object I presume we don't have to list them out in order to avoid returning the `data` blob.

Reply:
I don't think I understand what exactly is happening well enough to answer.
It's taking `FROM cas_2_applications` so unless we did want all the columns from a full application (including `data` we need to list out the columns we do want, that pertain to an `ApplicationSummary`? Or is the Java side able to take a whole application returned from the sql query and only return a summary?